### ISSUE TITLE
.die command doesn't kill FFMPEG processes and makes the bot crash

### ISSUE THREAD

**Issue by Midoriin** [Kwoth/NadekoBot#1329]
As title says, `.die` command doesn't kill FFMPEG processes which uses the same port and makes the bot crash with port errors. 

While bot is working
![While bot is working](http://midorina.ga/u/7c84f2.png)

When I kill it with .die
![When I kill it with .die](http://midorina.ga/u/b658e4.png)

Errors
![Errors](http://midorina.ga/u/aea155.png)

The only solution is killing the bot's session which also kills the zombie FFMPEG processes. So technically It's impossible to use auto restart If you assign a stable port to the clone. (I host 2 clones on a server, so I preferred assigning ports to them)

TL;DR The bot leaves zombie FFMPEG processes which is connected to the same port as It tries to reconnect and crashes.

**Comment by mroWEloh** [comment]
I have issues with the music module as well. When I queue like 50-ish songs and then save to a playlist, or just play them sometimes the bot will just randomly disconnect from voice chat and it won't allow some music commands to work, like `.s`, `.d`, `.n`, `.p`, etc. It also makes it impossible to rebind the bot to the voice channel without restarting the bot, which is super annoying if you have a currency event running because it ruins the event, and just in general annoying.

**Comment by Midoriin** [comment]
@mroWEloh You're totally right. There are a lot of issues. But recently, Kwoth has just released a new music module. We gotta try that out.

**Comment by Kwoth** [comment]
@Midoriin try it now
